src/Blueprint/Blueprint/Blueprint.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
>Project [](start = 1, length = 7)

After we migrate to CBT in our Blueprint repo, we'll be able to use thin projects like this :)